sdk/python/pulumi_kubernetes/yaml/yaml.py

```diff
@@ -13,7 +13,7 @@
 import requests
```

Comment:
Doesn't the change in namespace affect things like ConfigGroup usage etc. for users?

Reply:
It's still exported at the top level as before, so the path shouldn't matter. I didn't have to change any of the tests after this change.

Reply:
ah ok - missed that. LGTM